### PR TITLE
[PAY-634] On Ramp Design Fixes (Part 1)

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

See commits. Likely review by commit

- [PAY-641](https://linear.app/audius/issue/PAY-641/dont-show-the-disabled-button-state-when-you-start-the-coinbase-flow)
- [PAY-633](https://linear.app/audius/issue/PAY-633/modal-should-be-dismissable-when-an-error-occurs)
- [PAY-639](https://linear.app/audius/issue/PAY-639/lets-make-sure-we-format-our-token-name-as-dollaraudio)
- [PAY-642](https://linear.app/audius/issue/PAY-642/missing-the-ui-that-shows-the-units-of-the-quick-select-value-buttons)
- [PAY-643](https://linear.app/audius/issue/PAY-643/missing-the-ui-that-shows-the-currency-in-the-transaction-details)

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Locally against stage

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx [hunk 1]
```diff
@@ -105,17 +105,23 @@ export const AudioAmountPicker = ({
...
 
+  const id = useMemo(() => uniqueId(), [])
+
   return (
     <>
       {!isCustomAmountInputVisible ? (
         <AmountPreview amount={presetAmount} />
       ) : null}
       <RadioButtonGroup
+        aria-labelledby={`audioAmountPicker-label-${id}`}
```
Comment (dylanjeffers) [inline packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx:116]: Just wondering if we need a random Id? Are there gonna be multiple rendered?

**Review by dylanjeffers — COMMENTED** [review]
Looking cool

File: packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx:116]: Not currently, but better safe than sorry imo

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-onramp-design-fixes

**Review by sddioulde — APPROVED** [review]
lg

**Pull request merged**